src/sentry/middleware/integrations/tasks.py

```diff
@@ -60,11 +60,24 @@ def convert_to_async_slack_response(
     if not result["response"]:
         return
 
+    response_body = result["response"].content
+    if response_body == b"":
```

Comment:
If we just received an empty status response from the region, it's likely been handled without the need to proxy. Log it for debugging purposes only.